### ISSUE TITLE
Landing page background and foreground colors do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by rmjordas** [projectunic0rn/mentorship-ui#313]
Low-contrast text is difficult or impossible for many users to read. [Learn more](https://dequeuniversity.com/rules/axe/2.2/color-contrast?application=lighthouse).

Refer to the [Lighthouse report](https://lighthouse-dot-webdotdevsite.appspot.com/lh/html?url=https://www.mentrship.com/) for the list of failing elements.
